### PR TITLE
APIGOV-22478 April RelNotes

### PR DESCRIPTION
**Pull request by lbadenhop**
Thank you for your contribution to the Amplify-Central repo.

## Describe the changes

Enter a brief description of your changes here to communicate to the maintainers what you changed and why.

## Checklist for contributors

Before submitting this PR, please make sure:

* [ ] You have read the [contribution guidelines](https://axway-open-docs.netlify.com/docs/contribution_guidelines/)
* [ ] You have signed the [Axway CLA](https://cla.axway.com/)
* [ ] You have verified the technical accuracy of your change
* [ ] You have verified that your change does not expose any sensitive information (passwords, keys, etc.)
* [ ] You have followed the [Markdown guidelines](https://axway-open-docs.netlify.com/docs/contribution_guidelines/writing_markdown/)  (unless this is is a Netlify CMS contribution)

_Put an x in the boxes that apply. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your change._

### CODE REVIEW

**Comment by lbadenhop** [comment]
@cbordier @robertengaxway Please add content to pages.

File: content/en/docs/amplify_relnotes/20220430_foundry_relnotes.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+date: 2022-04-29
+---
+
+The Product Foundry enables providers to build API products by linking multiple assets to create a business capability and monetize them by defining subscription plans with different charging models.
+
+## New features and enhancements
+
+The following new features and enhancements are available in this update:
+
+* Ownership and sharing. When creating or editing a product, it is now possible to assign one owner team of the product and share it with other teams. The owning team is the sole that can edit/updated/delete the product. The shared with team will have a read access of the product.
```
Comment (robertengaxway) [inline content/en/docs/amplify_relnotes/20220430_foundry_relnotes.md:14]: Here is how I worded the Asset Catalog team ownership and sharing capability:
Ownership and sharing. When creating or editing a product, it is now possible to assign one owner team of the product and share it with other teams. The owning team and your role determine if you can edit/updated/delete the product. The shared with team will have read access to the product at a minimum.

Comment (cbordier) [reply content/en/docs/amplify_relnotes/20220430_foundry_relnotes.md]: I like that.

**Review by cbordier — APPROVED** [review]
I approve the PR.
@lbadenhop please proceed

**Comment by lbadenhop** [comment]
@cbordier @robertengaxway Please review my changes before I update the _index.md with the content from the Release Notes (not sure all my changes are correct--specifically the Asset Catalog Fixed issues).

**Review by robertengaxway — APPROVED** [review]
I have reviewed all files.  The index needs to text updates but other than that it looks good to me.

**Comment by lbadenhop** [comment]
@cbordier Please review the index before I merge. Thanks!

**Pull request merged**